app/src/main/java/com/kickstarter/libs/utils/SharedFunctions.kt

```diff
@@ -61,3 +65,20 @@ fun checkoutProperties(
         .shippingAmount(shippingAmount)
         .build()
 }
+
+fun <T : Relay> encodeRelayId(relay: T): String {
```

Comment:
encodeRelay is used on `Long` types, if you wanna extract it please do so in an Extension function for the `Long` class not into this class. 

This  `SharedFunctions` has been accepted as an exception for a concrete use case we had where the type using the functions was either `Backing` or `CheckoutData`, but should not be extended further that that.